### PR TITLE
Stomping tantrum smack down fix

### PR DESCRIPTION
**Pull request by pacmanboss256**

### CODE REVIEW

File: data/moves.ts [hunk 1]
```diff
@@ -16238,7 +16239,7 @@ export const Moves: {[moveid: string]: MoveData} = {
 		secondary: {
 			dustproof: true,
 			chance: 100,
-			onHit(target) {
+			onHit(target, source, move) {
```
Comment (Karthik99999) [inline data/moves.ts:16242]:
```suggestion
			onHit(target) {
```
no need to add `source` and `move` when you're not using them (and when they weren't there in the first place)

**Review by Zarel — APPROVED** [review]
Looks safe enough. I like it. @Marty-D?

**Comment by Marty-D** [comment]
Shouldn't this just be part of `cancelMove()`? I can't imagine subsequent Smack Downs on already smacked down Fly/Bounce users don't behave like this.

**Comment by pacmanboss256** [comment]
Would this affect other moves that use `cancelMove()` or do they all not trigger stomping tantrum

**Comment by Zarel** [comment]
Uses of `cancelMove` that may be relevant:

- Gravity canceling the second turn of Bounce, Fly, or Sky Drop
- Smack Down canceling the second turn of Bounce or Fly
- Pursuit moving itself up in priority if someone tries to switch
- Quash moving a move down in priority in Gen 7 and earlier

Quash and Pursuit can be refactored if they don't match. So the only question left is Gravity.

**Comment by Zarel** [comment]
I would be very surprised if Gravity worked differently from Smack Down, though. So sure, let's put it inside `cancelMove` and change Pursuit/Quash to manually rearrange queue order.

**Comment by DaWoblefet** [comment]
Yeah, to be explicit, Fly being interrupted by Gravity doesn't double Stomping Tantrum.

**Comment by pacmanboss256** [comment]
not sure how to change pursuit, and quash doesnt use `cancelMove()`

**Comment by pacmanboss256** [comment]
ignore the commit message branch names got messed up in vscode

**Pull request closed without merging**